docs/source/torch_environment_variables.rst

```diff
@@ -0,0 +1,20 @@
+.. _torch_environment_variables:
+
+Torch Environment Variables
+===============================
+PyTorch uses environment variables to control various settings, such as which
+gloo device to use, the number of threads used for parallelism, and the number
```

Comment:
I think this first sentence could highlight some more important ones, like "such as whether to print the C++ stacktrace when an error is hit, ..."